### PR TITLE
Fixed #2153 - fixes url of labels for search filtering

### LINKED ISSUE #2153: Redirect is not performed when clicking "Popular labels" from a search that returned no results
URL: https://staging.webcompat.com/issues?page=1&per_page=50&state=open&stage=all&sort=created&direction=desc
Browser / Version: Firefox Nightly Mobile 60.0a1 (2018-02-18), Firefox Nightly Desktop 60.0a1 (2018-02-16), Chrome Desktop 64.0.3282.167, Chrome Mobile 64.0.3282.137 
Operating System: Windows 10 Pro, Google Pixel (Android 8.0.0) - 1080 x 1920 pixels (~441 ppi pixel density)

**Steps to Reproduce:**
1. Navigate to https://staging.webcompat.com/issues?page=1&per_page=50&state=open&stage=all&sort=created&direction=desc
2. Tap the "Search" field and type in a string that returns no results (e.g. Testtesttest).
3. After error message is displayed, observe suggestions - popular labels (e.g. android/chrome/firefox/ie).
4. Tap one of the popular labels and observe behavior (e.g. chrome).

**Expected Behavior:**
Redirect to specific page is performed
(e.g. https://webcompat.com/issues?page=1&per_page=50&state=open&stage=all&sort=created&direction=desc&q=label%3Abrowser-chrome).  

**Actual Behavior:**
Nothing happens, no redirect is performed.

**Note**
1. Affected area: 
```html
<p class="issue-labels">
      Please check your spelling or try a more general search. 
	<br>
      You can also search for popular labels like: 
		<a class="label label-browser-android" href="#" title="Labels : android" data-remotename="browser-android">android</a>
		<a class="label label-browser-chrome" href="#" title="Labels : chrome" data-remotename="browser-chrome">chrome</a>
		<a class="label label-browser-firefox" href="#" title="Labels : firefox" data-remotename="browser-firefox">firefox</a>
		<a class="label label-browser-ie" href="#" title="Labels : ie" data-remotename="browser-ie">ie</a>
</p>
```

Watchers:
@softvision-sergiulogigan
@softvision-oana-arbuzov

screenshot-1519043167090

### PR DESCRIPTION
**Pull request by zoepage**
r? @softvision-oana-arbuzov 
r? @miketaylr

### CODE REVIEW

**Comment by zoepage** [comment]
adding @magsout as reviewer to this PR

File: webcompat/templates/list-issue/issuelist-issue.jst [hunk 1]
```diff
@@ -26,7 +26,7 @@
 <article class="grid-row">
   <div class="grid-cell issue x3">
     <h1 class="headline-1 js-no-results">Sorry, no results were found.</h1>
-     <p class="issue-labels">
+     <p class="issue-labels js-Issue-label">
```
Comment (miketaylr) [inline webcompat/templates/list-issue/issuelist-issue.jst]: What's the purpose of this class here?

Comment (zoepage) [reply webcompat/templates/list-issue/issuelist-issue.jst]: this class gives context to the label links. it grabs `data-remotename`, adds the correct path and creates a link. 

though this, we can use the labels as a filter.

Comment (miketaylr) [reply webcompat/templates/list-issue/issuelist-issue.jst]: roger that, thanks.

**Comment by miketaylr** [comment]
Actually, let's merge now and file a follow up for the test (since behavior didnt' really change, and the test has always been missing).

**Pull request merged**